### ISSUE TITLE
Segmentation of oranges isn't proper

### ISSUE THREAD

**Issue by Ayush12062000** [Ayush12062000/Orange-Fruit-Recognition-Using-Image-Segmentation#4]
So far what problem I found in the whole Process is the dataset, I guess since the dataset is self-made I think it has some errors with the Image-mask pair.

**Comment by nqa112** [comment]
Hi, have you modified the dataset ? I ran it and dataset is still incorrect.

**Comment by Ayush12062000** [comment]
No, I haven't modified the dataset. The dataset available is very less to run Unet for segmentation on it. I'll suggest you to create more annotations for Oranges and if the data size is decent Unet will work fine.

@nqa112
